app.js

```diff
@@ -5,16 +5,23 @@ require('dotenv').load({ silent: true })
 const glob = require('glob')
 const express = require('express')
 const bodyParser = require('body-parser')
+const bunyanMiddleware = require('bunyan-middleware')
+
+const logger = require('./lib/logger')
 
 const captureRaw = (req, res, buffer) => { req.raw = buffer }
 
 const app = express()
 app.use(bodyParser.json({ verify: captureRaw }))
+// bunyanMiddleware gives us request id's and unique loggers per incoming request,
+// for satefy reasons we don't want to include the webhook GitHub secret in logs
```

Comment:
ATM whitelisting is not implemented https://github.com/tellnes/bunyan-middleware#options. We could ofc open a PR if we really need it. Got any other sensitive headers we should avoid in the logs?